File: 3396-valid-word/3396-valid-word.kt
class Solution {
    fun isValid(word: String): Boolean {
        if (word.length < 3) return false

        var v = 0;
        var c = 0;

        for (x in word) {
            when {
                x.isLetter() -> {
                    if (x in listOf('a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U')) {
                        ++v;
                    } else {
                        ++c;
                    }
                }
                !x.isDigit() -> {
                    return false
                }
            }
        }
        return v>=1 && c>=1;
    }
}

File: 3396-valid-word/3396-valid-word.py
class Solution:
    def isValid(self, word: str) -> bool:
        if len(word) <3:
            return False
        v,c=0,0
        for i in word:
            if i.isalpha():
                if i in "aeiouAEIOU":
                    v+=1
                else:
                    c+=1
            elif not i.isdigit():
                return False
        return v>=1 and c>=1